code/__DEFINES/mobs.dm

```diff
@@ -434,7 +434,7 @@
 #define DOOR_CRUSH_DAMAGE 15 //the amount of damage that airlocks deal when they crush you
 
 #define HUNGER_FACTOR 0.05 //factor at which mob nutrition decreases
-#define ETHEREAL_CHARGE_FACTOR 0.8 //factor at which ethereal's charge decreases per second
+#define ETHEREAL_CHARGE_FACTOR (0.8 KILO WATTS) //factor at which ethereal's charge decreases per second
```

Comment:
should be renamed from factor to amount, since i can see where the confusion came from

Reply:
yeah, makes sense, did think about it buuut ended up forgetting about it